content/custom-decorators.md

```diff
@@ -210,7 +210,7 @@ You can then use this custom `@Auth()` decorator as follows:
 
 ```typescript
 @Get('users')
-@Auth('admin')
+@Auth(['admin'])
```

Comment:
the non-array version is correct. The `Auth`'s `role` parameter is an array of strings, not an array of arrays